core/templates/services/svg-sanitizer.service.spec.ts

```diff
@@ -80,7 +83,8 @@ describe('SvgSanitizerService', () => {
   });
 
   it('should check for invalid base64 images', () => {
-    expect(svgSanitizerService.isValidBase64Svg(invalidBase64data)).toBe(false);
+    expect(svgSanitizerService.isValidBase64Svg(invalidBase64data))
+      .toBe(false);
```

Comment:
Also, this can be in one line.
```suggestion
    expect(svgSanitizerService.isValidBase64Svg(invalidBase64data)).toBe(false);
```

Reply:
Done